admin/index_m.js

```diff
@@ -0,0 +1,111 @@
+// This will be called by the admin adapter when the settings page loads
+function load(settings, onChange) {
+    // example: select elements with id=key and class=value and insert value
+    if (!settings) return;
+    $('.value').each(function () {
+        let $key = $(this);
+        let id = $key.attr('id');
```

Comment:
Codacy has a fix for the issue: ['id' is never reassigned. Use 'const' instead.](https://app.codacy.com/gh/MK-2001/ioBroker.go-e/pullRequest?prid=7592349)
```suggestion
        const id = $key.attr('id');
```